resources/views/layouts/master.blade.php

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
```

Comment:
Delete this file. Look at layouts/app file on master branch. Copy and paste here and rename this file to make it work.